Movies-Selection/Movies-Selection/Moduls/MovieModule/View/MovieTableView.swift

```diff
@@ -58,12 +58,8 @@ final class MovieTableView: UITableViewController {
     }
 
     override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        let detailsView = DetailsFilmTableView()
-        let movieApiServices = MovieAPIServices()
-        let detailsPresenter = DetailsPresenter(view: detailsView, detailsApiServisec: movieApiServices)
-        detailsPresenter.movieID = presenter.movies[indexPath.row].id
-        detailsView.presenter = detailsPresenter
-        navigationController?.pushViewController(detailsView, animated: true)
+        let movieID = presenter.movies[indexPath.row].id
+        presenter.showDetailsModule(movieID: movieID)
```

Comment:
```suggestion
        presenter.showDetailsModule(movieID: movieID)
```
лучше не писать show и тд вью не знает что будет дальше лучше didSelect и в presenter.showDetailsModule(index: indexPath.row) передай просто индекс  а внутри перезентера уже возьми movies[index].id